wiki/snapshot.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Snapshots
+layout: wiki
+---
+
+# Overview
+Snapshots are a snapshot of the chain at a particular blockheight.
+Gridcoin snapshots are created by trusted developers and they allow newer users to get into sync faster than syncing from 0.
+
+# Issues
+* Snapshots are not validated by the wallet.
+* Snapshots are an inherently centralised method of getting into sync.
+* Users may mistakenly install a malicious or corrupted snapshot. 
+If enough users install the bad snapshot then the integrity of the chain could be compromised.
+
+# Using snapshots
+## Starting from 5.0.0
+Snapshots can now be downloaded and applied automatically by the wallet.
+
+It is accessible from File -> Snapshot download
+
+## Manual method
+1. [Download the snapshot](https://snapshot.gridcoin.us/snapshot.zip)
+2. [Verify the checksum (IMPORTANT)](https://snapshot.gridcoin.us/snapshot.zip.sha256)
+3. Delete the following folders: `txleveldb` & `accrual` from your Gridcoin data directory.
+4. Delete the `blk****.dat` files from your Gridcoin data directory.
+5. Extract the snapshot zip into your Gridcoin data directory.
+6. Restart the wallet and get into sync.
+
+# Alternatives
```

Comment:
I might add something here about how to create your own snapshots for yourself instead of relying on an external one